notebooks/405-paddleOCR-webcam/405-paddleOCR-webcam.ipynb

```diff
@@ -0,0 +1,466 @@
+   "metadata": {},
+   "source": [
+    "# PaddleOCR with OpenVINO\n",
+    "\n",
+    "This demo shows how to run PPOCR model on OpenVINO natively. Instead of exporting the PaddlePaddle model to ONNX and then convert to the Intermediate Representation (IR) format through OpenVINO model optimizer, we can now read directly from the Paddlepaddle Model without any conversions. [PaddleOCR](https://github.com/PaddlePaddle/PaddleOCR) is an ultra-light OCR model trained with Paddlepaddle deep learning frame work, aims to create multilingual and practical OCR tools. \n",
+    "\n",
+    "The paddleOCR pre-trained model used in the demo refer to the \"Chinese and English ultra-lightweight PP-OCR model (9.4M)\". More open-sourced pre-trained models could be downloaded at [PaddleOCR Github](https://github.com/PaddlePaddle/PaddleOCR)  or [PaddleOCR Gitee](https://gitee.com/paddlepaddle/PaddleOCR). Working pipeline of the paddleOCR is as follows:\n",
+    "\n",
+    "<img align='center' src= \"https://raw.githubusercontent.com/yoyowz/classification/master/images/paddleOCR_pipeline.png\" alt=\"drawing\" width=\"1000\"/>\n",
+    "\n",
+    "> Note: _To use this notebook with a webcam, you need to run the notebook on a computer with a webcam. If you run the notebook on a server, the webcam will not work. You can still do inference on a video._\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0e5a53f7-e1c5-4aca-879f-da2dd081b989",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Run Paddle Detection with OpenVINO"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f9486a04-b8bb-4bf5-9e13-845f2143a71b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "import cv2\n",
+    "import numpy as np\n",
+    "import paddle\n",
+    "import math\n",
+    "import time\n",
+    "import collections\n",
+    "from PIL import Image\n",
+    "\n",
+    "from openvino.runtime import Core,Dimension\n",
+    "from IPython import display\n",
+    "import copy\n",
+    "\n",
+    "sys.path.append(\"../utils\")\n",
+    "import notebook_utils as utils\n",
+    "import pre_post_processing as processing"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ee4ea41d-18a8-4914-b367-d5717111d8e8",
+   "metadata": {},
+   "source": [
+    "### Load the models for PaddleOCR\n",
+    "\n",
+    "PaddleOCR includes two parts of deep learning models, text detection and text recognition. Pre-trained models used in the demo are stored in the \"model\" folder. Other pre-trained models for PaddleOCR could be download at [PaddleOCR Github](https://github.com/PaddlePaddle/PaddleOCR)  or [PaddleOCR Gitee](https://gitee.com/paddlepaddle/PaddleOCR).\n",
+    "\n",
+    "Only a few lines of code are required to run the model. First, we create an Inference Engine. Then we read the network architecture and model weights from the .pdmoel and .pdiparams files to load onto the CPU."
```

Comment:
Could you explain that .pdmodel and .pdiparams are the files used by Paddle Paddle. This is new in the repo.